### ISSUE TITLE
Warning log: `ruff format` is a work-in-progress, subject to change at any time, and intended only for experimentation.

### ISSUE THREAD

**Issue by ancalita** [astral-sh/ruff#10845]
I'm replacing black formatter with ruff formatter and when reformatting the codebase with `ruff format` command, I keep seeing this warning:
```
warning: `ruff format` is a work-in-progress, subject to change at any time, and intended only for experimentation.
```

Could you please confirm if the formatter is stable and not likely to be removed in the future? If so, please consider this warning log since it's very confusing and makes users reluctant to use.

**Comment by AlexWaygood** [comment]
Hi! Would you mind pasting the output of running `ruff --version`?

**Comment by ancalita** [comment]
@AlexWaygood sure, this is the result:
```
ruff 0.0.291
```

**Comment by AlexWaygood** [comment]
Yeah, as @153957 says, ruff 0.0.291 is quite an old version of Ruff :-) If you use `ruff>=0.2`, the message should go away.

With regards to your broader question on the stability of the formatter: we describe the stability of the formatter in our versioning policy, which you can find here: https://docs.astral.sh/ruff/versioning/. In a nutshell: we promise to do our utmost to ensure that Ruff's "stable" formatting style will never change in a patch release -- this means that as long as your code contains valid syntax, bumping Ruff's version from e.g. 0.3.5 to 0.3.6 shouldn't result in the formatter reformatting your code. (If you bump Ruff from 0.3.6 to 0.4.0, however, it _might_ reformat your code.)

**Comment by ancalita** [comment]
Nice, thank you for the quick responses 🙌🏻

**Comment by AlexWaygood** [comment]
No worries! I'll close this for now, but let me know if you have any further questions :-)
